### PR TITLE
[MNT]Remove `.env` file (we shouldn't be pushing the `.env` file

### PR DESCRIPTION
**Pull request by fnhirwa**
### Description

This PR removes the `.env` file present in the repository as it is not a good practice to push changes to `.env` file.

Will first need to inspect the issue that this commit https://github.com/jdb78/pytorch-forecasting/commit/038b0c30c45866b83fd568a2069aeff42187f920 was resolving as it is when the file was added.

Once I am sure that this PR doesn't break anything we can go ahead with the review.

### CODE REVIEW

**Comment by fkiraly** [comment]
The "MPS" line might be linked to this?
https://github.com/jdb78/pytorch-forecasting/issues/1596

It does not look like anything is failing, anyway.

**Comment by fkiraly** [comment]
See here for the actual failure: https://github.com/jdb78/pytorch-forecasting/pull/1633

**Comment by fkiraly** [comment]
might I suggest to create a branch off this PR, and merge it with #1633, just to see what happens?

**Comment by fnhirwa** [comment]
This is device handling related, I am going to open an issue to fix this generally and start working on a Pull Request to fix this.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -79,7 +79,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-13, windows-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```
Comment (fkiraly) [inline .github/workflows/test.yml]: I would suggest, let's not reconfigure the mac VM in this PR, but only separately - now the CI also fails.

Comment (fnhirwa) [reply .github/workflows/test.yml]: Yeah we should isolate that I am experimenting this in https://github.com/jdb78/pytorch-forecasting/pull/1648
Still haven't found the working solution.

Comment (fkiraly) [reply .github/workflows/test.yml]: could you split and keep only the env removal in this PR? That looks fine by itself.

**Comment by fkiraly** [comment]
ready for merge?

**Comment by fnhirwa** [comment]
Yes it is ready to merge.

**Review by fkiraly — APPROVED** [review]

**Pull request merged**